src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx

```diff
@@ -260,25 +260,22 @@ BOOL APIENTRY DllMain(HMODULE hModule,
     return TRUE;
 }
 
-// TODO remove pHttpContext from the CreateApplication call.
 HRESULT
 __stdcall
 CreateApplication(
     _In_  IHttpServer        *pServer,
-    _In_  IHttpContext       *pHttpContext,
-    _In_  PCWSTR             pwzExeLocation,
+    _In_  IHttpApplication   *pHttpApplication,
```

Comment:
I thought the HttpContext was used in the request handler config to check websocket support. Can you check what we do for websockets?

Reply:
You moved websocket check to CreateHandler where we have a context.